### PR TITLE
Add code to connect to PubNub service and render some UI during the connection process

### PR DESCRIPTION
**Pull request by clbond**

### CODE REVIEW

File: services/connection-state.js [hunk 1]
```diff
@@ -0,0 +1,6 @@
+export const ConnectionState = {
+  Idle: 'Idle',
+  Connecting: 'Connecting',
+  Connected: 'Connected',
+  Failed: 'Failed',
+};
```
Comment (joehandlebar) [inline services/connection-state.js]: maybe this should be in 'constants' and not 'services' ?

File: services/configuration.js [hunk 2]
```diff
@@ -0,0 +1,7 @@
+export const channels = ['ReactChat'];
+
+export const configuration = {
+  publishKey: 'pub-c-199f8cfb-5dd3-470f-baa7-d6cb52929ca4',
+  subscribeKey: 'sub-c-d2a5720a-1d1a-11e6-8b91-02ee2ddab7fe',
+  ssl: true,
+};
```
Comment (joehandlebar) [inline services/configuration.js]: this too (?)

Comment (clbond) [inline services/configuration.js]: Sure. Just a sec

**Comment by joehandlebar** [comment]
:shipit:

**Pull request merged**